src/lib/tooltip/tooltip.ts

```diff
@@ -524,10 +527,10 @@ export class TooltipComponent {
   tooltipClass: string|string[]|Set<string>|{[key: string]: any};
 
   /** The timeout ID of any current timer set to show the tooltip */
-  _showTimeoutId: number;
+  _showTimeoutId: number | null;
```

Comment:
Optional: you could use zero (so that you don't need the `| null`) since `setTimeout` always returns a positive integer.